github-templates/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+### Summary
+
+### Expected behaviour
+
+### Actual behaviour
+
+### Steps to reproduce
```

Comment:
A good point @benlast . I'll try a second pass on this to incorporate feature requests.